README.rst

```diff
@@ -72,23 +73,21 @@ To install from source (available on GitHub in `dwavesystems/dwave-cloud-client`
 
 .. _`dwavesystems/dwave-cloud-client`: https://github.com/dwavesystems/dwave-cloud-client
 
-.. installation-end-marker
-
 
 License
--------
+=======
 
 Released under the Apache License 2.0. See `<LICENSE>`_ file.
 
 
 Contributing
-------------
+============
 
-Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+Ocean's `contributing guide <https://docs.dwavequantum.com/en/latest/ocean/index_ocean_sdk.html>`_
```

Comment:
Contributing guide should be on a separate page, not index/front.

Let's put it at: `https://docs.dwavequantum.com/en/latest/ocean/contributing.html`.

Reply:
Updated